src/components/CollectionInfoContent.js

```diff
@@ -2,11 +2,12 @@ import React, { useState } from "react";
 import { Form, Layout, Button, Typography, Input, Modal, message } from "antd";
 import { Row, Col } from "react-flexbox-grid";
 import styled from "styled-components";
-import FaceMask from "../Icons/FaceMask";
-import Gloves from "../Icons/Gloves";
-import HandSanitizer from "../Icons/HandSanitizer";
-import Suit from "../Icons/Suit";
+import { ReactComponent as FaceMask } from "../Icons/FaceMask.svg";
+import { ReactComponent as Gloves } from "../Icons/Gloves.svg";
+import { ReactComponent as Suit } from "../Icons/Suit.svg";
+import { ReactComponent as HandSanitizer} from '../Icons/disinfectant.svg';
```

Comment:
Is there a reason why you made this change?

Having SVG's as React components gives opportunities to allow props to be passed into them

Reply:
React cannot resolve style property "enable-background" in svg. It is still an [open ticket](https://github.com/reactjs/react-magic/issues/63)

Reply:
Good catch! We can revisit this when/if we decide to use props